source/includes/read-write-configuration.c

```diff
@@ -0,0 +1,70 @@
+
+// start-client-write-concern
```

Comment:
Consider adding `main` to make example compilable. That may help validate future changes. Here is an [example refactor of this file](https://github.com/kevinAlbs/c-bootstrap/blob/915d7d86ed457d4c3e5f354ec78844f219a29bda/investigations/doc-example/main.c).

Reply:
Thanks for that example! I'll start mirroring this type of format in future c pages. Updated!